models/products-model.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+const ProductsSchema =  new Schema({
+  img: 
+      { data: Buffer, contentType: String }
+  , 
+  name : {
+    type : String , 
+    required : true
+  } , 
+  description : {
+    type : String ,
+    required : true
+  }
+    , 
+  price : {
+    type : Number ,  
+    required : true
+  } , 
+   
+  inStock : {
+    type : Boolean ,  
+    required : true
+  } , 
+  fastDelivery : {
+    type : Boolean ,  
+    required : true
+  } ,
+    quantity : {
+    type : Number ,  
+    required : true
+  } 
+}
+)
+
+const Products = mongoose.model('Products' , ProductsSchema , 'products')
```

Comment:
make model as product , then in mongoDB it will reflect as products .Here you are making model as products , so it will reflect as productss . And I think it is not good.

Reply:
no himanshu , it is reflecting as products only